ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -198,7 +198,7 @@ open class TableViewCell: UITableViewCell, TokenizedControlInternal {
         return self
     }
 
-    @objc private func themeDidChange(_ notification: Notification) {
+    @objc func themeDidChange(_ notification: Notification) {
```

Comment:
@harrieshin wanted this particular change to go from main->fluent2_colors, so I'll spin up a PR for that one, and you can cherry-pick that over.